docs/rollup.config.js

```diff
@@ -3,11 +3,12 @@ import replace from '@rollup/plugin-replace';
 import commonjs from '@rollup/plugin-commonjs';
 import svelte from 'rollup-plugin-svelte';
 import babel from '@rollup/plugin-babel';
-import alias from '@rollup/plugin-alias';
 import { terser } from 'rollup-plugin-terser';
 import config from 'sapper/config/rollup.js';
-import autoPreprocess from 'svelte-preprocess';
 import pkg from './package.json';
+
```

Comment:
What is this separation for?

Reply:
The imports above are from the template, it's just easier to compare with the template to make sure we're still up-to-date on the best practices